client/main/variableeditor.js

```diff
@@ -221,15 +221,16 @@ const EditorWidget = Backbone.View.extend({
         this.$el.empty();
         this.$el.addClass('silky-variable-editor-widget');
 
-        this.$title = $('<input class="silky-variable-editor-widget-title" type="text">').appendTo(this.$el);
+        this.$title = $('<input class="silky-variable-editor-widget-title" type="text" maxlength="50">').appendTo(this.$el);
```

Comment:
you could handle it at the cython level, but if you wanted to handle it at the C++ level, i'd do it like this:

    int length = strlen(name);
    char *chars = _mm->allocate<char>(length);

    if (length > 63)  // 63 is probably a better limit than 50
        length = 63;
    std::memcpy(chars, name, length);
    chars[length] = '\0';

**however**, because we're using UTF-8, we might chop a character off part way through, so i would handle this at the cython level. which would be:

    name = name[:63]

oh wait, this won't work either :/ python will give you 63 characters as per your request, but because of UTF-8, that might resolve to more than 63 bytes. [this](http://www.gossamer-threads.com/lists/python/python/751333) should do it:

    name = name.encode("utf8")[:63].decode("utf8", "ignore")